### DISCUSSION TITLE
Is it possible to model DAG in Cloudsimplus?

### DISCUSSION THREAD

**Discussion by LeeBoonKong** [manoelcampos/cloudsim-plus#372] [Q&A] [answered]
Is it possible to create a series of task that resembles a Direct Acyclic Task model in Cloudsimplus, schedule and execute them in VM?

**Comment by manoelcampos** [comment, marked as answer]
Yes, it is. You have to define the relationships between your tasks (NetworkCloudlets) and define the packet transfer between them. Check the [example here](https://github.com/cloudsimplus/cloudsimplus-examples/blob/master/src/main/java/org/cloudbus/cloudsim/examples/network/applications/NetworkVmsExampleWorkflowApp.java). It does exactly what I'm saying.

**Comment by deRemo** [comment]
Hi @manoelcampos, your example link now is located [here](https://github.com/cloudsimplus/cloudsimplus-examples/blob/master/src/main/java/org/cloudbus/cloudsim/examples/network/applications/NetworkVmsExampleWorkflowApp.java). Speaking of: I'm unable to get a proper outcome from such test, as the VMs go in destruction phase right at the start of the simulation. I tried `setVmDestructionDelayFunction()` on broker, `setShutDownDelay()` and `setShutDownBeginTime()` on VM, but nothing. Can you please help me configure the test to work properly? Thanks
